conference/react-js/src/components/AddNewOrganizer/AddNewOrganizer.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import Dialog from 'material-ui/Dialog';
+import RaisedButton from 'material-ui/RaisedButton';
+import MenuItem from 'material-ui/MenuItem';
+import SelectField from 'material-ui/SelectField';
+import TextField from 'material-ui/TextField';
+import { conferencesNames, organizerInvite } from '../../constants/backend-url';
+
+class AddNewOrganizer extends Component {
+  constructor(prop) {
+    super(prop);
+    this.state = {
+      open: false,
+      conferenceName: '',
+      userEmail: '',
+      conferenceList: [],
+    };
+  }
+
+  componentDidMount() {
+    axios.get(conferencesNames)
+      .then(({ data }) => {
+        this.setState({
+          conferenceList: data,
+        });
+      });
+  }
+
+  setMenuItems = () => (
```

Comment:
as soon as this piece of code returns menu items list it will be much better to use `getMenuItems` name